extensions/Microsoft.AspNetCore.Runtime.SiteExtension/Microsoft.AspNetCore.Runtime.SiteExtension.csproj

```diff
@@ -10,14 +10,26 @@
 
   <ItemGroup>
     <Content Include="applicationHost.xdt" />
+    <Content Include="scmApplicationHost.xdt" />
     <Content Include="install.cmd" />
     <Content Include="$(OutputPath)\Microsoft.Web.Xdt.Extensions.dll" PackagePath="content" />
 
     <Content Include="$(DotnetHomeDirectory)\$(RuntimeArch)\**\*.*" Condition="$(DotnetHomeDirectory) != ''" PackagePath="content\%(RecursiveDir)%(FileName)%(Extension)" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(RuntimeArch)' == 'x86'">
```

Comment:
nit: Why not RuntimeArchitecture?

Reply:
RuntimeArchitecture might affect other msbuild targets and we don't want that.